_pkgdown.yml

```diff
@@ -24,6 +24,8 @@ articles:
     - create-individual
     - create-run-population
     - observed-data
+    - data-combined
+    - data-combined-plotting

```

Comment:
should we add a separate group for these 2 vignettes (e.g. "Plotting")?

Reply:
Yes good idea.

Reply:
A new group now.